### ISSUE TITLE
Unregister for key presses while the ConsoleMenu is open.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Games#53]
While the console menu is open the script should stop receiving key presses. This is because if the player opens the console they may accidently hit one of the menu prompt keys.

**Comment by TheWhiteCollarPlayers** [comment]
What does this mean? Do you mean that while the console is open the game code should stop receiving key presses? I.e. if the PC opens the console then they cant accidently hit one of the menu prompt keys?

**Comment by Scrivener07** [comment]
Yes exactly.

**Comment by Scrivener07** [comment]
Fixed with commit dc72ebc

**Comment by Scrivener07** [comment]
I fixed it by checking is the console menu is opened in the key press event.
[Difference Report](https://github.com/Scrivener07/FO4_Games/commit/dc72ebc3f38a20aa6b1dce313618041e3a7f9928#diff-9ad1ab2cc08124d7bffe9a7fa5e010e9R86)

```
	Event OnKeyDown(int keyCode)
 		If (!UI.IsMenuOpen("Console"))
 			...
```
